src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolumeHelperUnoptimized.h

```diff
@@ -358,7 +358,7 @@ static void vtkUnoptimizedInsertSlice(vtkPlusPasteSliceIntoVolumeInsertSlicePara
   {
     for (int idY = inExt[2]; idY <= inExt[3]; idY++, inPtr += inIncY, importancePtr += impIncY)
     {
-      for (int idX = inExt[0]; idX <= inExt[1]; idX++, inPtr += numscalars, importancePtr += impIncX)
+      for (int idX = inExt[0]; idX <= inExt[1]; idX++, inPtr += numscalars, importancePtr += 1)

```

Comment:
I'm not sure if it makes any difference in performance, but it's a hot loop so maybe it's better to write importancePtr++